tests/test_models.py

```diff
@@ -0,0 +1,173 @@
+from datetime import datetime, timedelta
+
+from django.conf import settings
+
+import pytest
+
+from .factories import (ValueFactory, ValuesetFactory, MetadataFactory,
+                        MetadataValuesFactory, ValuesetValuesFactory,
+                        ProjectFactory, ProjectMetadataFactory, NominatorFactory,
+                        URLFactory, NominatedURLFactory)
+
+
+pytestmark = pytest.mark.django_db
+
+
+class TestValue:
+    def test_unicode(self):
+        value = ValueFactory.build()
+
+        assert unicode(value) == value.value
+
+
+class TestValueSet:
+    def test_unicode(self):
+        valueset = ValuesetFactory.build()
+
+        assert unicode(valueset) == valueset.name
+
+
+class TestMetadata:
+    def test_unicode(self):
+        metadata = MetadataFactory.create()
+
+        assert unicode(metadata) == metadata.name
+
+
+class TestMetadataValues:
+    def test_unicode(self):
+        metadata_values = MetadataValuesFactory.create()
+
+        assert unicode(metadata_values) == '{0} ({1})'.format(
+            metadata_values.metadata, metadata_values.value)
+
+
+class TestValuesetValues:
+    def test_unicode(self):
+        valueset_values = ValuesetValuesFactory.create()
+
+        assert unicode(valueset_values) == '{0} ({1})'.format(
+            valueset_values.valueset, valueset_values.value)
+
+
+class TestProject:
+    def test_unicode(self):
+        project = ProjectFactory.build()
+
+        assert unicode(project) == project.project_slug
+
+    def test_nomination_message_before_nomination_window(self):
+        project = ProjectFactory.build(
```

Comment:
This is being duplicated in almost every test in the test case. Have you considered moving it into a fixture?